### ISSUE TITLE
Will support for 60-second video generation be available in the future?

### ISSUE THREAD

**Issue by leetraman822** [kijai/ComfyUI-EasyAnimateWrapper#6]
Hi, thanks for your great node! 
Does this support long video generation? I noticed that the maximum frame adjustment in the comfyui node is 144 frames (6 seconds), while the official gradio provides 1440 frames (60 seconds, though I haven't tested it yet). I have used the comfyui node you created, which quickly generated the video, whereas the official gradio's generation speed is very slow and inefficient. Your node has greatly improved the efficiency.

**Comment by kijai** [comment]
I don't know why it would be 1440 when the model specs say it only supports up to 144, might be a typo? Though I've never tried more myself either.

**Comment by leetraman822** [comment]
Okay.😊 Using your node, generating a 32-frame video with a 512x512 model takes only about 1-2 minutes, and even the 144 frames with the 768x768 model is remarkably fast! In contrast, generating 144 frames with the 768x768 model on the official gradio is painfully slow—it took over 40 minutes just to show the progress bar after I clicked the generate button, which was quite horrifying. Though, I did have a swift experience with the demo on Hugging Face Space as well, leading me to wonder if there's something missing in my environment or not properly installed. Frankly, had I first attempted video generation using the local gradio instead of your ComfyUI node, I might have lost all hope in my computer's capabilities. My setup includes an RTX 3060 12GB GPU and 16GB of system memory. 
All in all, this is an incredible node, and I'm truly appreciative of your work.😊
